### DISCUSSION TITLE
How to find eigenfrequencies of a steel grid construction (FEM)

### DISCUSSION THREAD

**Discussion by Hyppias** [mathnet/mathnet-numerics#1097] [Q&A]
I have the stiffness matrix [K] and the mass matrix [M] of a steel grid structure.
I should be able to get the eigenfrequencies of the structure with those matrices.

Theoretically they follow from

|[K] - v[M]| = 0,

, an (eigenproblem) where v (really omega squared) is the vector of eigenvalues (from which you can derive the frequencies).

How can I do that with what MathNet has to offer?
It has methods to get the eigenfrequencies of a matrix, but I have two of them: [K] and [M], each being, say, 100 x 100.

**Comment by jkalias** [comment]
You can try the method described in https://arxiv.org/pdf/1903.11240, section 7.1 (disclaimer: I never tried it myself)

Let me know if it worked, I’m also curious.
